pkg/build/build.go

```diff
@@ -109,6 +109,16 @@ func (b *Builder) generateCombustionScript() error {
 	return nil
 }
 
+func (b *Builder) writeCombustionFile(contents string, filename string) error {
```

Comment:
Can we reuse this in `copyCombustionFile()`?

Reply:
Probably. I thought about deprecating the copy one but I'm not 100% convinced yet that everything will be embeddable. I'll keep it in mind for a future refactor.